docs/apache-airflow-providers-apache-druid/connections/druid.rst

```diff
@@ -0,0 +1,120 @@
+.. _druid_integration:
```

Comment:
Same comment as for https://github.com/apache/airflow/pull/35244#discussion_r1375316519

We need to align the connections first